platform/src/pages/opinions/new/input_opinion.rs

```diff
@@ -1,13 +1,445 @@
 use dioxus::prelude::*;
-use dioxus_translate::Language;
+use dioxus_logger::tracing;
+use dioxus_translate::{translate, Language};
+
+use crate::{
+    components::{
+        icons::{Search, Switch, UploadFile},
+        upload_button::UploadButton,
+    },
+    pages::opinions::new::{controller::Controller, i18n::OpinionNewTranslate},
+};
+
+#[cfg(feature = "web")]
+use dioxus::html::HasFileData;
 
 #[derive(Props, Clone, PartialEq)]
 pub struct InputOpinionProps {
     lang: Language,
 }
 
+#[derive(Clone, PartialEq)]
+pub enum DocumentTabType {
+    DirectUpload,
+    Import,
+}
+
+#[derive(Props, Clone, PartialEq)]
+pub struct ConnectProjectTranslate {
+    research_project_linkage: String,
+    research_project_linkage_description: String,
+    research_selection: String,
+}
+
+#[derive(Props, Clone, PartialEq)]
+pub struct UploadDocumentTranslate {
+    upload_document: String,
+    upload_document_description: String,
+    direct_upload: String,
+    import_from_data_management: String,
+    upload_file_description: String,
+    load_file: String,
+    upload_file_warning: String,
+    title: String,
+    document_type: String,
+    field: String,
+    purpose_of_use: String,
+    source: String,
+    authority: String,
+    last_modified_date: String,
+    form: String,
+}
+
+#[derive(Props, Clone, PartialEq)]
+pub struct InputIntroductionTranslate {
+    enter_introduction: String,
+    introduction_description: String,
+    select_field: String,
+    enter_title_hint: String,
+    enter_description_hint: String,
+}
+
 #[component]
 pub fn InputOpinion(props: InputOpinionProps) -> Element {
-    let _props = props;
-    rsx! { "implement this ui" }
+    let translates: OpinionNewTranslate = translate(&props.lang);
+    rsx! {
+        div { class: "flex flex-col w-full justify-start items-start",
+            div { class: "font-medium text-[16px] text-[#000000] mb-[10px]",
+                "{translates.essential_information}"
+            }
+            InputIntroduction {
+                lang: props.lang.clone(),
+                i18n: InputIntroductionTranslate {
+                    enter_introduction: translates.enter_introduction.to_string(),
+                    introduction_description: translates.introduction_description.to_string(),
+                    select_field: translates.select_field.to_string(),
+                    enter_title_hint: translates.enter_title_hint.to_string(),
+                    enter_description_hint: translates.enter_description_hint.to_string(),
+                },
+            }
```

Comment:
```suggestion
            InputIntroduction {
                lang: props.lang,
                i18n: InputIntroductionTranslate {
                    enter_introduction: translates.enter_introduction.to_string(),
                    introduction_description: translates.introduction_description.to_string(),
                    select_field: translates.select_field.to_string(),
                    enter_title_hint: translates.enter_title_hint.to_string(),
                    enter_description_hint: translates.enter_description_hint.to_string(),
                },
            }
```
일단 lang 을 클론하지 않아도 burrow 가 안될거에요. 이게 props 로 묶여 있어서 어떨지 모르겠지만, 분리된 경우는 burrow 가 안됩니다.
i18n 을 이렇게 넘기지 말고, 각 컴포넌트에서 호출해서 써주시고, translate! 는 컴포넌트 단위로 분리해주세요.
https://github.com/biyard/democrasee/blob/main/packages/main-ui/src/pages/i18n.rs 여기 코드 참고해주세요.